frontend/src/services/dataService.js

```diff
@@ -0,0 +1,18 @@
+import axios from 'axios';
+
+const URL = axios.create({
+    withCredentials: true,
+    baseURL: 'http://localhost:8000',
```

Comment:
Are you using 8000 as the backend port?

PS: If possible can you look into proxying the request to backend so that it won't be neccessary to have the baseURL we can just make a request using `/events/ping`. The port to proxy to should be `http://localhost:2121`

Reply:
Got a http://localhost:2121 proxy working. Should we merge this PR into add-event-form and then open a new PR for the proxy?

Reply:
> Got a http://localhost:2121 proxy working. Should we merge this PR into add-event-form and then open a new branch/PR to deal with the proxy?

Just to clarify. I merge this to add event form - and then you PR again to add event form?

Reply:
That's what I'm thinking, so that the proxy stuff can be on a separate branch that gets tested and then merged in its own PR.

Reply:
Ok I'm fine with that. Will merge now.